### PR TITLE
feat(uptime): Add models to support regions on a subscription

### PR DESCRIPTION
**Pull request by wedamija**
We need to be able to keep track of which regions are associated with a subscription. For now, we'll automatically fill these in based on any configured regions. In the future we may have different scheduling modes where users can specify the region, but for now the behaviour will be consistent.

### CODE REVIEW

File: src/sentry/uptime/models.py [hunk 1]
```diff
@@ -107,6 +108,28 @@ class Meta:
...
 
 
+@region_silo_model
+class UptimeSubscriptionRegion(DefaultFieldsModel):
+    __relocation_scope__ = RelocationScope.Excluded
+
+    uptime_subscription = FlexibleForeignKey("uptime.UptimeSubscription")
+    # The url to check
+    slug = models.CharField(max_length=255)
+    name = models.CharField(max_length=255)
```
Comment (JoshFerge) [inline src/sentry/uptime/models.py:118]: what are slug and name? how is that related to the url comment?

Comment (wedamija) [reply src/sentry/uptime/models.py:118]: Oops, the comment was a copy/paste leftover.

slug is the slug for the region, name is the more human readable name

**Review by JoshFerge — APPROVED** [review]

**Comment by wedamija** [comment]
Actually, going to add a m2m table here
